src/libs/CardUtils.ts

```diff
@@ -36,6 +36,17 @@ function getMonthFromExpirationDateString(expirationDateString: string) {
     return expirationDateString.substring(0, 2);
 }
 
+/**
+ * @param card
+ * @returns number
+ */
```

Comment:
can you please add plain english explanation here as it would not be clear for anyone why we use this.

Please put it on two lines as needed, dont see it really well in this suggestion view
```suggestion
/**
 * Ensure to sort physical Expensify cards first, no matter what their cardIDs are. This way ensures the Expensify Combo Card detail is rendered correctly, because we will always use the cardID of the physical card from the combo card duo.
 * @param card
 * @returns number
 */
```